cms/static/sass/elements/_modal-window.scss

```diff
@@ -579,28 +579,33 @@
   .wrapper-modal-window-bulkpublish-subsection,
   .wrapper-modal-window-bulkpublish-unit,
   .course-outline-modal {
-    .exam-time-list-fields,
-    .exam-review-rules-list-fields {
-      margin: 0 0 ($baseline/2) 0;
+    .no-descriptive-text {
+      display: block;
+      margin-bottom: ($baseline/4);
     }
+
     .list-fields {
       .field-message {
-        @extend %t-copy-sub2;
+        @extend %t-copy-sub1;
         color: $gray-d1;
+        margin-bottom: ($baseline/4);
       }
       label {
-        @extend %t-title7;
+        @extend %t-title6;
+        font-weight: font-weight(semi-bold);
       }
+
       .field {
         display: inline-block;
         vertical-align: top;
         @include margin-right($baseline/2);
         margin-bottom: ($baseline/4);
 
         label {
-          @extend %t-strong;
+          @include font-size(16);
```

Comment:
Overriding font-size and weight for consistency with label change above. Otherwise, font size is 14.